### PR TITLE
Hotfix 22.06 release for Pin Cuda Python Packages

### PR DESCRIPTION
**Pull request by caryr35**
This 22.06 Hotfix was to pin cuda-python max version to 11.7.0 to python packages

### CODE REVIEW

File: _notices/rgn0028.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+notice_topic: Release Change
+notice_rapids_version: "v22.06"
+notice_created: 2022-07-07
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-07-07
+---
+
+## Overview
+
+The `cuda-python` package dependency specification was pinned to a maximum version of `11.7.0` for several RAPIDS libraries.
```
Comment (aravenel) [inline _notices/rgn0028.md:29]:
```suggestion
RAPIDS v22.06 was hotfixed to pin the `cuda-python` package dependency to a maximum version of `11.7.0` for several RAPIDS libraries.
```

File: _notices/rgn0028.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Rationale
+
+`cuda-python` version `11.7.1` was recently released, which switched its dependency on the [cudatoolkit](https://anaconda.org/nvidia/cudatoolkit) package to the [cuda-toolkit](https://anaconda.org/nvidia/cuda-toolkit) package. This dependency change had unintended consequences for several RAPIDS libraries.
```
Comment (aravenel) [inline _notices/rgn0028.md:33]:
```suggestion
`cuda-python` version `11.7.1` was recently released, which switched its dependency on the [cudatoolkit](https://anaconda.org/nvidia/cudatoolkit) package to the [cuda-toolkit](https://anaconda.org/nvidia/cuda-toolkit) package. This dependency change had unintended consequences for several RAPIDS libraries, including blocking installation for some users.
```

File: _notices/rgn0028.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Impact
+
+All users are recommended to use the latest `22.06.xx` version of all RAPIDS libraries.
```
Comment (aravenel) [inline _notices/rgn0028.md:37]: Need to update the version number--believe this is only 22.06 hotfix to date, so should be 22.06.01.

**Review by aravenel — CHANGES_REQUESTED** [review]

File: _notices/rgn0028.md (hunk 3 above)
Comment (caryr35) [reply _notices/rgn0028.md]: updated version to 22.06.01

**Review by aravenel — APPROVED** [review]

**Pull request merged**